content/applications/inventory_and_mrp/inventory/shipping/setup/bpost.rst

```diff
@@ -0,0 +1,109 @@
+=================
+Bpost integration
+=================
+
+Set up the *Bpost* shipping connector in Odoo to manage Bpost shipments to clients directly within
+Odoo. To configure it, complete these steps:
+
+#. Create a Bpost account.
+#. Get the :ref:`Account ID and passphrase <inventory/shipping/Bpost-account>`.
+#. Set up the shipping method in Odoo.
+
+Upon completion, it is possible to calculate the cost of shipping, based on package size and weight,
+have the charges applied directly to a Bpost business account, and automatically print Bpost
+tracking labels through Odoo.
+
+.. seealso::
+   - :doc:`third_party_shipper`
+   - :doc:`delivery_method`
+   - :doc:`dhl_credentials`
+   - :doc:`ups_credentials`
+
+Bpost account setup
+===================
+
+To begin, go to the `Bpost website <https://parcel.bpost.be/en/home/business>`_ to create, or log
+into, the company's Bpost business account. When creating the Bpost account, have the company's VAT
+number and mobile phone number ready.
+
+Follow the website's steps to complete registration, and sign up for shipping services.
+
+After completing the setup, get the Bpost account ID and passphrase, by navigating to the
+:guilabel:`Shipping Manager` menu item.
+
+.. image:: bpost/shipping-manager.png
+   :align: center
+   :alt: Show header menu on Bpost website, highlighting the *Shipping Manager* option.
+
+.. _inventory/shipping/bpost-account:
+
+On the :guilabel:`Shipping Manager` page, go to the :guilabel:`Admin` tab, then the
+:guilabel:`General Settings` tab, to find the :guilabel:`Account ID` and :guilabel:`Passphrase`
+needed to configure Odoo's shipping method.
+
+.. image:: bpost/credentials.png
+   :align: center
+   :alt: In the *Admin* tab, show the Account ID and Passphrase.
+
+Configure bpost shipping method
+===============================
+
+With those necessary credentials, configure the Bpost shipping method in Odoo by going to
+:menuselection:`Inventory app --> Configuration --> Shipping Methods`.
+
+On the :guilabel:`Shipping Methods` page, click :guilabel:`Create`.
+
+In the :guilabel:`Provider` field, select :guilabel:`Bpost` from the drop-down menu. Doing so
+reveals the :guilabel:`Bpost Configuration` tab at the bottom of the form, where the Bpost
+credentials can be entered.
+
+For details on configuring the other fields on the shipping method, such as :guilabel:`Delivery
+Product`, refer to the :doc:`Configure third-party carrier <third_party_shipper>` documentation.
+
+.. note::
+   To generate Bpost :doc:`shipping labels <../operation/labels>` through Odoo, ensure the
+   :guilabel:`Integration Level` option is set to :guilabel:`Get Rate and Create Shipment`.
+
+In the :guilabel:`Bpost Configuration` tab, complete the following fields:
+
+- :guilabel:`Bpost Account Number` (required field): enter the company's unique :ref:`account number
+  <inventory/shipping/bpost-account>` from the Bpost website.
```

Comment:
Suggestion to use "ID" instead of number, since that is the field name on Bpost :)
```suggestion
- :guilabel:`Bpost Account Number` (required field): enter the company's unique :ref:`account ID
  <inventory/shipping/bpost-account>` from the Bpost website.
```